modules/uid2IdSystem.js

```diff
@@ -183,29 +71,24 @@ export const uid2IdSubmodule = {
    * @returns {uid2Id}
    */
   getId(config, consentData) {
-    const result = getIdImpl(config, consentData);
+    const mappedConfig = {
+      apiBaseUrl: config?.params?.uid2ApiBase ?? UID2_BASE_URL,
+      paramToken: config?.params?.uid2Token,
+      serverCookieName: config?.params?.uid2Cookie ?? config?.params?.uid2ServerCookie,
+      storage: config?.params?.storage ?? 'localStorage',
+      clientId: UID2_CLIENT_ID,
+      internalStorage: ADVERTISING_COOKIE
+    }
+    _logInfo(`UID2 configuration loaded and mapped.`, mappedConfig);
+    const result = Uid2GetId(mappedConfig, storage, _logInfo, _logWarn);
```

Comment:
not passing consent at all to this function is a bit concerning; as we discussed in chat, you have to recheck it, you can't assume it hasnt changed since the token was generated.

Reply:
It now bails out prior to this if GDPR applies. For the similar piece of code in the EUID module, the same - it checks consent and bails out prior to this call.